pytorch_cnn_trainer/engine.py

```diff
@@ -188,6 +194,9 @@ def val_step(
     top1_m = utils.AverageMeter()
     top5_m = utils.AverageMeter()
     cnt = 0
+    losses_v = []
```

Comment:
Again same comment from `train_step`. I'm already returning metrics on every `step` of training.
I want metrics for every `epoch` how does logging here solve the issue?